server/src/main/java/dev/slimevr/Main.java

```diff
@@ -13,7 +13,9 @@
 
 public class Main {
 
-	public static final String VERSION = "0.5.1";
+	public static final String VERSION = BuildConfig.GIT_VERSION_TAG.isEmpty()
+		? BuildConfig.GIT_VERSION_TAG
+		: BuildConfig.GIT_COMMIT_HASH;
```

Comment:
This looks like the wrong order to me
```suggestion
	public static final String VERSION = !BuildConfig.GIT_VERSION_TAG.isEmpty()
		? BuildConfig.GIT_VERSION_TAG
		: BuildConfig.GIT_COMMIT_HASH;
```